Tests/test_SeqIO_convert.py

```diff
@@ -102,28 +104,40 @@ def compare_record(old, new, truncate=None):
             raise ValueError("'%s' vs '%s'" % (old.seq, new.seq))
         else:
             raise ValueError("'%s...' vs '%s...'" % (old.seq[:100], new.seq[:100]))
-    if "phred_quality" in old.letter_annotations \
-       and "phred_quality" in new.letter_annotations \
-       and old.letter_annotations["phred_quality"] != new.letter_annotations["phred_quality"]:
-        if truncate and [min(q, truncate) for q in old.letter_annotations["phred_quality"]] == \
-                        [min(q, truncate) for q in new.letter_annotations["phred_quality"]]:
+    if (
+        "phred_quality" in old.letter_annotations
+        and "phred_quality" in new.letter_annotations
+        and old.letter_annotations["phred_quality"]
+        != new.letter_annotations["phred_quality"]
+    ):
+        if truncate and [
+            min(q, truncate) for q in old.letter_annotations["phred_quality"]
+        ] == [min(q, truncate) for q in new.letter_annotations["phred_quality"]]:
```

Comment:
I don't like how this if-statement (and the similar one below) look after black. Can you try adding brackets round the equality bit, which I think would help? I am expecting that would put the two list-comprehensions each on their own line.

Reply:
It doesn't look better. if I add brackets or split it to align, black keeps bringing the operator, equal, to the front of the list-comprehension.

if I do this:

        if truncate and (
            [min(q, truncate) for q in old.letter_annotations["phred_quality"]]
        ) == (
                [min(q, truncate) for q in new.letter_annotations["phred_quality"]]
        ):

Black does this:

        if truncate and (
            [min(q, truncate) for q in old.letter_annotations["phred_quality"]]
        ) == ([min(q, truncate) for q in new.letter_annotations["phred_quality"]]):

If I do this:

        if truncate and [
            min(q, truncate) for q in old.letter_annotations["phred_quality"]
        ] == [
            min(q, truncate) for q in new.letter_annotations["phred_quality"]
        ]:

Black does this:

        if truncate and [
            min(q, truncate) for q in old.letter_annotations["phred_quality"]
        ] == [min(q, truncate) for q in new.letter_annotations["phred_quality"]]:

No improvement.

Reply:
Oh well, thanks for trying.